index.html

```diff
@@ -1135,6 +1160,169 @@ <h3>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Application icons and images
+      </h2>
+      <p>
+        Application icons can be used by the user agent in multiple places,
```

Comment:
Nit: Maybe just talk about "icons" or do we want to define "application icons" specifically.